discord-api.js

```diff
@@ -15,24 +15,35 @@ const isUserARadar = async (access_token) => {
     return Boolean(guild);
 };
 
-const exchangeCodeForToken = async (code) => {
-    const params = new URLSearchParams();
-    params.append('client_id', process.env.CLIENT_ID);
-    params.append('client_secret', process.env.CLIENT_SECRET);
-    params.append('grant_type', 'authorization_code');
-    params.append('code', code);
-    params.append('redirect_uri', 'https://fkipongejlaaachjiaipijmmnhcacbca.chromiumapp.org');
-    params.append('scope', 'identify guilds');
+const exchangeCodeForToken = async (redirect_uri) => {
+    try {
 
-    const authResponse = await fetch(`https://discord.com/api/oauth2/token`, {
-        method: 'POST',  
-        body: params,
-        headers: {
-            'Content-Type': 'application/x-www-form-urlencoded',
-            'Accept': 'application/json'
-        }
-    });
-    return  authResponse.json();
+        const redirect = redirect_uri.split('?')[0];
```

Comment:
It wouldn't be a bad idea to use URLSearchParams here like you're doing from line 24 also.